.github/scripts/fetch_compare_version.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+
+import re
+import shlex
+import subprocess
+import sys
+
+
+def fetch_version_from_code():
+    command = 'egrep "set *\(WAMR_VERSION" build-scripts/config_common.cmake'
```

Comment:
I wonder if we can avoid using egrep and use native python libraries/functions to get the version? That would get us one step closer to have it working on Windows.